src/styles/block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  list-style: none;
+}
+
+.stars__star {
+  background-image: url(../../src/images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 > .stars__star:nth-child(1n + 5) {
```

Comment:
all stars modifiers should work correctly (not just --4)

Reply:
and try to use here [cycle](https://sass-lang.com/documentation/at-rules/control/for)